frontend/src/api/types.ts

```diff
@@ -30,37 +113,76 @@ export type AnalysisResults = {
   df_edge: { Node1: string; Node2: string }[];
   df_edge_sym: { Node1: string; Node2: string }[];
   df_probs: {
-    "Class-Label": "P" | "N" | "U";
+    Rank: number;
     Entrez: string;
-    "Known/Novel": "Known" | "Novel";
+    Symbol: string;
     Name: string;
     Probability: number;
-    Rank: number;
-    Symbol: string;
+    "Known/Novel": "Known" | "Novel";
+    "Class-Label": "P" | "N" | "U";
   }[];
   df_sim: {
+    Rank: number;
     ID: string;
     Name: string;
-    Rank: number;
     Similarity: number;
   }[];
 };
 
-export type Species =
-  | "Human"
-  | "Mouse"
-  | "Fly"
-  | "Zebrafish"
-  | "Worm"
-  | "Yeast";
+/** backend format to frontend format */
```

Comment:
Fairly cut and dry here. Bunch of schema definitions and functions to convert between them.  Backend results converted into formats that are a bit easier for the frontend to work with.